### ISSUE TITLE
Load carts that are version > 8

### ISSUE THREAD

**Issue by Sterophonick** [picolove/picolove#77]
Most of the carts nowadays are version later than 8, and it is really starting to cause the emulator some issues with functionality. Most carts that I own are unplayable, due to an incorrect version. Please add support for it sometime in the future.

**Comment by bluss** [comment]
Is it known what is missing/what needs to be researched to update?

**Comment by Shoozza** [comment]
Not really, there are more and more things getting added every release so the list grows.
Would probably be good to have some list/documentation for that.
Alternatively test carts for a specific pico-8 versions to verify working functionality?
